cucumber/api/features/policy_versions_retention.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Limit the number of policy versions
+
+  Each update of a policy creates a policy version record and policy log records. This feature limits
+  the number of versions in the database to default value of 20 version, older versions are deleted.
+
+  Background:
+    Given I am the super-user
+    And I successfully POST "/policies/cucumber/policy/root" with body:
+    """
+      - !policy policy_test_version
+    """
+
+  Scenario: I update a policy multiple times so it exceeds the default policy versions limit and get the default
+            limited number of versions in response when I retrieve the policy resource
+    Given I save my place in the log file
+    And I successfully POST 21 times "/policies/cucumber/policy/policy_test_version" with body:
+    """
+      - !user bob
+    """
+    Then the HTTP response status code is 201
```

Comment:
This is not what you test here imo. you test that we remove the 21st version so i'd change this to be Given (and) and let the next Then be the first one.
```suggestion
    And the HTTP response status code is 201
```

Reply:
anyway you shouldn't have 2 `Then`s